### PR TITLE
tools/zep_dispatch: document foren6 usage

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

I noticed that the latest addition from #16879 was only documented in the PR description, not the README of the tool itself.

### Testing procedure

[Read](https://github.com/RIOT-OS/RIOT/blob/49b0cbddd80617b952c9bfcb181195ece3b797c3/dist/tools/zep_dispatch/README.md) (and try following the instructions to see if they make sense).

### Issues/PRs references

follow up on #16879

### CODE REVIEW

File: dist/tools/zep_dispatch/README.md [hunk 1]
```diff
@@ -44,3 +44,103 @@ refer to the file `example.topo` for an example.
...
+You should see a view of the DODAG with all nodes in the network as they send packets.
+
+![](https://user-images.githubusercontent.com/1301112/144511776-3a2d7072-8162-40dc-911f-dfe476d01112.png)
+
+#### RIOT setup
+
+A proper simulated network will need a border router and some mesh nodes.
+The `gnrc_border_router` and `gnrc_networking` can serve as a starting point for those.
+
+By default the border router example will start the ZEP dispatcher with a flat toplology, that is
```
Comment (fjmolinas) [inline dist/tools/zep_dispatch/README.md:89]:
```suggestion
By default the border router example will start the ZEP dispatcher with a flat topology, that is
```

File: dist/tools/zep_dispatch/README.md [hunk 2]
```diff
@@ -44,3 +44,103 @@ refer to the file `example.topo` for an example.
 There can only be as many nodes connected to the dispatcher as have been named in
 the topology file.
 Any additional nodes that try to connect will be ignored.
+
+Network visualisation
```
Comment (fjmolinas) [inline dist/tools/zep_dispatch/README.md:48]:
```suggestion
Network visualization
```

File: dist/tools/zep_dispatch/README.md [hunk 3]
```diff
@@ -44,3 +44,103 @@ refer to the file `example.topo` for an example.
...
+
+    make graph
+
+target. This sends a USR1 signal to the `zep_dispatch` process to generate a graph in
+Graphviz format.
+
+
+### Foren6
+
+For real-time visualisation of the (RPL) network, the [Foren6](https://cetic.github.io/foren6/)
```
Comment (fjmolinas) [inline dist/tools/zep_dispatch/README.md:61]:
```suggestion
For real-time visualization of the (RPL) network, the [Foren6](https://cetic.github.io/foren6/)
```

**Comment by fjmolinas** [comment]
49b0cbddd80617b952c9bfcb181195ece3b797c3 seems unrelated to the PR title

**Comment by benpicco** [comment]
> 49b0cbd seems unrelated to the PR title

Yea I just realized that when writing to restart the sniffer, I noticed that doesn't actually work if a sniffer was attached to the dispatcher previously. I think it makes more sense that always the latest sniffer is used instead of the first one.

**Review by fjmolinas — APPROVED** [review]
ACK

**Pull request merged**